scrapers/PureTaboo.yml

```diff
@@ -2,21 +2,21 @@ name: puretaboo
 sceneByURL:
   - action: scrapeXPath
     url:
-      - https://www.puretaboo.com/en/video
+      - www.puretaboo.com/en/video
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //div[@class="styles_Rms8xL5kg6"]/h2
+      Title: //meta[@name="twitter:title"]/@content
       Image: //video[@class="vjs-tech"]/@poster
```

Comment:
the comment should go here
```yaml
      Image: //video[@class="vjs-tech"]/@poster
      #//meta[@name="twitter:image"]/@content #for image in 4K
```
instead at the end of the file
For some of the scernes i tried the resulting image seems to be 720p though not 4k ( for those missing the 4k tag i guess)

Reply:
Well my selector have a better quality than the video poster, not always 4K.
The only true difference is sometimes image is different like this:
``` 
//video: https://transform.gammacdn.com/movies/69325/69325_01/previews/2/239/top_1_960x544/69325_01_01.jpg?width=960&height=543&enlarge=true
//meta: https://images03-fame.gammacdn.com/movies/69325/69325_01/previews/2/239/top_1_resized/69325_01_01.jpg
```

This is why i keep the `//video` as principal because sometime he got the name of scene in the screenshot. But `//meta` can be good option too. I just give the user all possibility.